tests/testsuite/build_script.rs

```diff
@@ -149,6 +151,46 @@ fn custom_build_env_vars() {
         p.cargo("build").arg("--features").arg("bar_feat"),
         execs().with_status(0),
     );
+
+    // Test passing linker from .cargo/config to env var
+    if cross_compile::disabled() { return; }
```

Comment:
Could this be split into a separate test rather than tacked onto the previous test?

Reply:
Yeah let's go ahead and split out the test